server/src/pages/shot/view.js

```diff
@@ -388,6 +394,7 @@ class Body extends React.Component {
 
     return (
       <reactruntime.BodyTemplate {...this.props}>
+        { this.state.imageEditing ? this.renderEditor(clip) : null }
```

Comment:
Seems odd this would render the entire page, plus the editor...?  I'd expect it to render the editor *or* the normal page.